android/build.gradle

```diff
@@ -15,28 +24,54 @@ rootProject.allprojects {
     }
 }
 
+allprojects {
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+
+apply plugin: "com.android.library"
+apply plugin: "kotlin-android"
+
 android {
-    compileSdkVersion 34
     if (project.android.hasProperty("namespace")) {
-        namespace 'com.clevertap.clevertap_plugin'
+        namespace = "com.clevertap.clevertap_plugin"
     }
 
-    defaultConfig {
-        minSdkVersion 19
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    compileSdk = 34
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
     }
-    lintOptions {
-        disable 'InvalidPackage'
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8
     }
-    compileOptions {
-        sourceCompatibility = 1.8
-        targetCompatibility = 1.8
+
+    sourceSets {
+        main.java.srcDirs += "src/main/kotlin"
+        test.java.srcDirs += "src/test/kotlin"
+    }
+
+    dependencies {
+        implementation 'androidx.core:core-ktx:1.9.0'
+        testImplementation 'junit:junit:4.13.2'
+        api 'com.clevertap.android:clevertap-android-sdk:7.0.3'
+        compileOnly 'androidx.fragment:fragment:1.3.6'
+        compileOnly 'androidx.core:core:1.9.0'
     }
-}
 
-dependencies {
-    testImplementation 'junit:junit:4.13.2'
-    api 'com.clevertap.android:clevertap-android-sdk:7.0.3'
-    compileOnly 'androidx.fragment:fragment:1.3.6'
-    compileOnly 'androidx.core:core:1.9.0'
+    testOptions {
```

Comment:
We don't have any unit tests. Is this block needed?

Reply:
removing this.